charts/am-role-assignment-service/values.preview.template.yaml

```diff
@@ -28,16 +28,14 @@ java:
     ROLE_ASSIGNMENT_DB_OPTIONS: "?stringtype=unspecified"
     CCD_DATA_STORE_URL: http://ccd-data-store-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
-    # TODO: remove legacy LD env vars in DTSAM-789
+    # TODO: remove legacy LD env vars after DTSAM-492
```

Comment:
NB: remove legacy LD env vars after [DTSAM-492](https://tools.hmcts.net/jira/browse/DTSAM-492), i.e. #2334, is successfully deployed in PROD.

Reply:
Resolved in 84fa55d4c126a26450300a885c2cb261f022d8ec